flink-table/flink-table-planner-blink/src/test/java/org/apache/flink/table/planner/catalog/CatalogStatisticsTest.java

```diff
@@ -111,10 +119,62 @@ public void testGetStatsFromCatalogForCatalogTableImpl() throws Exception {
 				false);
 
 		alterTableStatistics(catalog);
-
 		assertStatistics(tEnv);
 	}
 
+	@Test
+	public void testGetPartitionStatsFromCatalog() throws Exception {
```

Comment:
please add some tests about UNKNOWN stats

Reply:
I can add others except unknown fields, because `CatalogColumnStatisticsDataBase` not support unknown now.

Reply:
https://issues.apache.org/jira/browse/FLINK-14663, and @zjuwangg is fixing it

Reply:
I think this blocker is test blocker, we can wait it until 1.10 release.

Reply:
BTW, the PR is: https://github.com/apache/flink/pull/10394